src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
```

Comment:
According to the Java style guide, the names of constants (static final fields) should be in uppercase. So `MAX_NUMBER` should be renamed to `MAX_NUMBER`.